feature_extraction.py

```diff
@@ -0,0 +1,55 @@
+import csv
+from nltk.tokenize import word_tokenize
+from nltk.corpus import stopwords
+from nltk import ngrams
+from collections import Counter
+
+bigramsDictionary = Counter()
+unigramsDictionary = Counter()
+bigramsProbability = {}
+
+def tolower(sentence):
+    for w in sentence:
+        sentence = w.lower()
+    return sentence
+
+def remove_nonletter(sentence):
+    filtered_sentence = []
+    for w in sentence:
+        if w.isalpha:
+            filtered_sentence.append(w)
+    return filtered_sentence
+
+def stopword_removal(sentence):
+    filtered_sentence =[]
+    stopWords = set(stopwords.words('english'))
+    for word in sentence:
+        if word not in stopWords:
+            filtered_sentence.append(word)
+    return filtered_sentence
+
+def getUnigrams(token):
+	unigramsDictionary.update(Counter(token))
+	return
+
+def getBigrams(token):
+    bigramsDictionary.update(Counter(token))
+    return
```

Comment:
```suggestion
def updateUnigrams(token):
	unigramsDictionary.update(Counter(token))

def updateBigrams(token):
    bigramsDictionary.update(Counter(token)) 
```
Here I would suggest you to not use `get`, since you are actually not returning anything. Instead, you are updating, which I think it's pretty self-explanatory. By the way, you can take advantage of Python implicit `None` returns in this case.

Reply:
I did not know that. Will keep in mind.